std/digest/digest.d

```diff
@@ -2,20 +2,34 @@ module std.digest.digest;
 
 import _newDigest = std.digest;
 
-// scheduled for deprecation in 2.077
-// See also: https://github.com/dlang/phobos/pull/5013#issuecomment-313987845
+// @@@DEPRECATED_2.084@@@
+deprecated("std.digest.digest is deprecated. Please use std.digest. Removal in 2.084. Removal in 2.084")
```

Comment:
You repeated the removal time.

Can we use a sentence that helps the user by explicitly telling them what to do? Like this

```d
deprecated("import std.digest instead of std.digest.digest. std.digest.digest will be removed in 2.084")
```

Reply:
> Can we use a sentence that helps the user by explicitly telling them what to do? Like this

Good idea. Done.